src/client/features/interactions/filter-menu.js

```diff
@@ -92,9 +98,10 @@ class InteractionsFilterMenu extends React.Component {
 
     //Slider listed under 'Visible Nodes' in the interaction viewer
     const slider = [
-      h("input",{type:"range",id:'selection-slider',min:0,max:maxSliderVal,step:0.0001,defaultValue:defaultSliderVal,
+      h("input",{type:"range",id:'selection-slider',min:0,max:1,step:0.0001,defaultValue:0.9,
```

Comment:
- use single quotes for strings 
- add whitespace after commas, brackets, and colons. 

```js
h('input', { type: 'range', id: 'selection-slider', min: 0, max: 1, step: 0.0001, defaultValue: 0.9, 
```